src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -0,0 +1,98 @@
+import { ChartType, Plugin } from "chart.js";
+import { chartFontColor } from "../../../../helpers/figures/charts/chart_common";
+import { Color } from "../../../../types";
+
+interface ChartShowValuesPluginOptions {
+  showValues: boolean;
+  background?: Color;
+  horizontal?: boolean;
+  callback?: (any) => any;
+}
+
+declare module "chart.js" {
+  interface PluginOptionsByType<TType extends ChartType> {
+    chartShowValuesPlugin?: ChartShowValuesPluginOptions;
+  }
+}
+
+/** This is a chartJS plugin that will draw connector lines between the bars of a Waterfall chart */
+export const chartShowValuesPlugin: Plugin = {
+  id: "chartShowValuesPlugin",
+  afterDatasetsDraw(chart: any, args, options: ChartShowValuesPluginOptions) {
+    if (!options.showValues) {
+      return;
+    }
+    const drawData = chart._metasets?.[0]?.data;
+    if (!drawData) {
+      return;
+    }
+    const ctx = chart.ctx;
+    ctx.save();
+
+    ctx.textAlign = "center";
+    ctx.textBaseline = "middle";
+    ctx.fillStyle = chartFontColor(options.background);
```

Comment:
I don't see a differente with or witout this. Am I blind ?

Reply:
Yes you are ...

Joke apart, you will see a difference in some case where we want the value to be "highlighted" over line/point or area
![image](https://github.com/odoo/o-spreadsheet/assets/91665701/a857eb7a-1465-4584-8c32-62960258bc64)